docs/fast_data/faq/bootstrapping-fast-data.md

```diff
@@ -0,0 +1,160 @@
+:::
+
+### What is the topic setup I should use to guarantee an high availability?
+
+A topic is distributed among the brokers of your cluster, with a parameter called _replication factor_: for critical messages, this value should be set to the number of brokers, meaning that the data of a topic will be stored inside all brokers.
+
+You can also specify how many replicas are needed to send acknowledgments on produced messages before committing, to guarantee the synchronization of the distributed data.
+
+This value can be set at the topic level with the parameter `min.insync.replicas` and is usually equal to `# of brokers - 1`. Critical flows may require a number of in-sync replicas equal to the number of brokers. 
+
+## Mongo Db
+
+### What should be the required resources to perform an Initial Load?
+
+The **Initial Load** is a manual operation made at runtime to initialize the projection's records with the most recent copy of the whole dataset from the system of record linked to it.
+
+This operation is carried out either the first time the system is boot up, when no records have ever been ingested, or whenever a schema change is deemed relevant enough to require reloading all the table content: for example, a new field is added to the schema of a table and it should be added retroactively to records that have already been ingested but that might not receive further updates in the future.
+
+The component that will perform receive the record, for example from a Change Data Capture (CDC), is the [Real-Time Updater (RTU)](/fast_data/realtime_updater.md).
+
+:::danger Before executing the Initial Load
+The RTU can be enabled to [execute strategies](/fast_data/the_basics.md#strategies), custom functions that from an ingestion record performs a set of queries to extract one or more projection change records that will be used to generate a Single View.
+
+Since an high number of ingestion messages will be handled by the RTU, remember **to disable strategy execution** during the Initial Load.
+
+This will avoid to perform additional MongoDB queries made by the strategies to generate projection changes records.
+:::
+
+:::tip Execution Time
+An initial load of `6.000.000 records` took around _10 minutes_, which is approximately a rate of `60k message/minute`: for this message rate, up until 70% of CPU from a single MongoDB core was used.  
+:::
+
+### What should be the required resources to perform a Single View Refresh?
+
+The **Single View Refresh** is an operation performed to update all the projection changes of a single view: this activity is needed during the first release of a single view and every time a new feature is released in the environment, since it will start the aggregation pipeline of the [**Single-View Creator**. (**SVC**)](/fast_data/single_view_creator.md)
```

Comment:
```suggestion
The **Single View Refresh** is an operation performed to update all the projection changes of a single view: this activity is needed during the first release of a single view and every time you need to add new fields also on older SVs, since it will start the aggregation pipeline of the [**Single-View Creator**. (**SVC**)](/fast_data/single_view_creator.md)
```